<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TimeTable</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="timeTableView">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>11</y>
      <width>1191</width>
      <height>781</height>
     </rect>
    </property>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>50</number>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
